Equilobe.DailyReport.Models/Storage/AtlassianWorklog.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace Equilobe.DailyReport.Models.Storage
+{
+    public class AtlassianWorklog
+    {
+        public long Id { get; set; }
+        public long InstalledInstanceId { get; set; }
+        public long AtlassianUserId { get; set; }
+        public long IssueId { get; set; }
+        public string IssueKey { get; set; }
+        public long JiraWorklogId { get; set; }
+        public string Comment { get; set; }
+        public DateTime CreatedAt { get; set; }
+        public DateTime UpdatedAt { get; set; }
```

Comment:
Can updatedAt be null? or this is also initialised when the worklog is created?

Reply:
It is initialized when the worklog is created